iphone/Maps/Core/Settings/MWMSettings.mm

```diff
@@ -92,6 +92,23 @@ + (void)setTheme:(MWMTheme)theme
 {
   if ([self theme] == theme)
     return;
+  if (@available(iOS 13.0, *)) {
+    UIUserInterfaceStyle style;
+    switch (theme) {
+      case MWMThemeDay:
+      case MWMThemeVehicleDay:
+        style = UIUserInterfaceStyleLight;
+        break;
+      case MWMThemeNight:
+      case MWMThemeVehicleNight:
```

Comment:
Is Outdoor theme related to these values? How is this enum used, and can it be simplified? Is it needed at all?

Reply:
1. Outdoor is not part of this enum. It is used only in Core managers.
<img width="376" alt="image" src="https://github.com/organicmaps/organicmaps/assets/79797627/03179253-f166-4521-8ee8-afb4c01b2845">

2. it is hard to say now is these cases MWMThemeVehicleNight/day are still useful... needs additional refactoring. 
Core enum contains them:
<img width="301" alt="image" src="https://github.com/organicmaps/organicmaps/assets/79797627/0a7fdbcc-f603-4bfc-a07a-a2d639ae4162">

Should I refactor these cases?

Reply:
_Kostyli_ detected, let's refactor them later.